modules/preview-react/pill/lib/PillIcon.tsx

```diff
@@ -21,7 +21,7 @@ export const PillIcon = createSubcomponent('span')({
 })<PillIconProps>(({size, icon, ...elemProps}, Element) => {
   return (
     <SystemIcon
-      marginInlineStart={`-${space.xxxs}`} // remove padding on the left from HStack
+      marginInlineStart={`-${space.xxxs}`} // remove excess margin from the start
```

Comment:
I wonder if this still needed, Hstack was using margin, but if we use Flex, there's no extra margin, I'll test this locally